magento/magento2
# Map modal widget file paths to aliases

### Description

In this [devdoc](http://devdocs.magento.com/guides/v2.0/javascript-dev-guide/javascript/custom_js.html), the outlined method of overriding or extending the modal widgets could not be done without duplicating the entire code of the respective widget. The proposed solution would allow the modal widgets to be extended and/or overridden rather than only overridden.

### Manual testing scenarios

The proposed solution should be easily verified as working by viewing any modal on a clean install. One example of where a modal can be opened is on the Customer Account -> Add Address page.

### Contribution checklist
